### DISCUSSION TITLE
0 internal and 0 external displays detected SEB error

### DISCUSSION THREAD

**Discussion by Tapasd** [SafeExamBrowser/seb-win-refactoring#182] [Q&A]
![Screenshot_2](https://user-images.githubusercontent.com/89982551/131847183-363cb7d9-2ef3-4881-8ecb-8dadfb12d735.png)

Heya, keep getting this error regarding 0 external and internal displays when im running my practice test links, anyone know how i can solve it? I am running SEB in VMware for your knowledge. Here's the log, not entirely sure if I've uploaded this correctly. First post on github
[2021-09-02_14h50m22s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7098981/2021-09-02_14h50m22s_Runtime.log)

**Comment by dbuechel** [comment]
If you are a student, then it should be pretty obvious that you're not supposed to run SEB in a virtual machine. If you're a teacher, then you'd need to allow validation failures of the display configuration in the configuration tool under tab "Security".

**Reply by Tapasd** [reply]
> 
> 
> If you are a student, then it should be pretty obvious that you're not supposed to run SEB in a virtual machine. If you're a teacher, then you'd need to allow validation failures of the display configuration in the configuration tool under tab "Security".

is there no way to fix it as a student? havent seen anyone with this same error, one with 0 detected screens

**Reply by dbuechel** [reply]
In short: no. Again, you're most likely not supposed to run SEB in a virtual machine as student...

**Comment by Filocava99** [comment]
@Tapasd you would have to modify the DisplayMonitor.ValidateConfiguration method to return a result with the property IsAllowed always true. I'm not gonna tell you further details though
